app/controllers/provider/moderation.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+    acceptedpage: 1,
```

Comment:
What are the purposes of these values?

Reply:
track the current page for each list type (pending, accepted or rejected)